### ISSUE TITLE
GPU Plugin turning off since latest update

### ISSUE THREAD

**Issue by Joee94** [runelite/runelite#16737]
### Type

GPU

### Operating System

MacOS

### OS Version

Monterey 12.5.1

### Bug description

GPU plugin won't turn on. Was working this morning. When trying to turn it on it turns itself off.

This is the error from the client.log
```2023-07-01 18:29:30 BST [Client] ERROR n.r.client.plugins.gpu.OpenCLManager - [LWJGL] cl_context_callback: [CL_INVALID_DEVICE] : OpenCL Error : clCreateCommandQueue failed: Unable to locate device 0x1024500 in context 0x6000027a6730.
2023-07-01 18:29:30 BST [Client] ERROR n.r.client.plugins.gpu.OpenCLManager - [LWJGL] cl_context_callback: [CL_INVALID_DEVICE] : OpenCL Error : clGetProgramBuildInfo failed: Device 0x1024500 not in context 0x6000027a6730
2023-07-01 18:29:30 BST [Client] ERROR n.r.client.plugins.gpu.GpuPlugin - Error starting GPU plugin
java.lang.RuntimeException: OpenCL error [-33]
	at net.runelite.client.plugins.gpu.OpenCLManager.checkCLError(OpenCLManager.java:571)
	at net.runelite.client.plugins.gpu.OpenCLManager.getProgramBuildInfoInt(OpenCLManager.java:542)
	at net.runelite.client.plugins.gpu.OpenCLManager.compileProgram(OpenCLManager.java:335)
	at net.runelite.client.plugins.gpu.OpenCLManager.compilePrograms(OpenCLManager.java:366)
	at net.runelite.client.plugins.gpu.OpenCLManager.init(OpenCLManager.java:133)
	at net.runelite.client.plugins.gpu.GpuPlugin.initProgram(GpuPlugin.java:575)
	at net.runelite.client.plugins.gpu.GpuPlugin.lambda$startUp$1(GpuPlugin.java:373)
	at net.runelite.client.callback.ClientThread.invokeList(ClientThread.java:119)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:101)
	at net.runelite.client.callback.Hooks.tick(Hooks.java:215)
	at client.ok(client.java:8713)
	at client.bz(client.java)
	at ba.bo(ba.java:382)
	at ba.run(ba.java:361)
	at java.base/java.lang.Thread.run(Unknown Source)
2023-07-01 18:29:30 BST [Client] ERROR n.r.client.callback.ClientThread - Exception in invoke
java.lang.NullPointerException: null
	at org.lwjgl.system.Checks.check(Checks.java:188)
	at org.lwjgl.opencl.CL10.clReleaseKernel(CL10.java:5760)
	at net.runelite.client.plugins.gpu.OpenCLManager.cleanup(OpenCLManager.java:146)
	at net.runelite.client.plugins.gpu.GpuPlugin.lambda$shutDown$2(GpuPlugin.java:455)
	at net.runelite.client.callback.ClientThread.lambda$invoke$0(ClientThread.java:49)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:62)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:47)
	at net.runelite.client.plugins.gpu.GpuPlugin.shutDown(GpuPlugin.java:429)
	at net.runelite.client.plugins.gpu.GpuPlugin.lambda$startUp$1(GpuPlugin.java:420)
	at net.runelite.client.callback.ClientThread.invokeList(ClientThread.java:119)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:101)
	at net.runelite.client.callback.Hooks.tick(Hooks.java:215)
	at client.ok(client.java:8713)
	at client.bz(client.java)
	at ba.bo(ba.java:382)
	at ba.run(ba.java:361)
	at java.base/java.lang.Thread.run(Unknown Source)```

### Screenshots or videos

_No response_

### RuneLite version

RuneLite 1.10.6 (launcher version 2.6.3) 

### Logs

[client.log](https://github.com/runelite/runelite/files/11926774/client.log)
[launcher.log](https://github.com/runelite/runelite/files/11926775/launcher.log)

**Comment by Felanbird** [comment]
Issue is being worked on in the discord

**Comment by Adam-** [comment]
fixed in ac48bba29a413f0b2ed68a60fbf8559244bc287b
